src/table/components/head/HeadCellMenu.tsx

```diff
@@ -66,10 +35,10 @@ export default function HeadCellMenu({
           ]
         : []),
     ],
-    [translator, isInteractionEnabled, isCurrentColumnActive, sortDirection, isDimension]
+    [translator, isDimension]
   );
 
-  return (
+  return menuItemGroups.length ? (
```

Comment:
Nit-picking but it might be easier to read the code of you exit early and return null when it is a measure instead. But that is mostly me not being a fan of long ternary operator statements

Reply:
hmm. yeah I think I'd prefer that as well. I will keep the spreading in the menuItemGroups array though, since we will need that as soon as we add more options.

Reply:
actually you can't early return, because then you get hooks called conditionally. This will be cleaned up soon anyway